### ISSUE TITLE
Plex Remote Access Not Utilizing Aggregated Connection

### ISSUE THREAD

**Issue by TipsTricksMore** [Ysurac/openmptcprouter#3580]
Hi,

I’ve been using OpenMPTCPRouter for over a year in my office to bond two 1 Gbit/s coaxial connections into a combined 2 Gbit/s downlink. My primary goal was to achieve a more reliable uplink of 2x 50 Mbit/s, which has successfully provided me with a steady 100 Mbit/s upload, and it’s been working great for my needs.

I supported this project with a tip a while back because it has been incredibly helpful. Thank you for all your hard work!

Recently, I’ve started hosting a Plex server on my NAS. I have some large video files with bitrates of around 80 Mbit/s that I’d love to stream at home, where I have a 1 Gbit/s connection. In theory, this should work perfectly, but I’ve noticed an issue: Plex seems to only utilize one of my uplinks, maxing out at 50 Mbit/s, which leads to buffering.

This behavior is puzzling because when I run speed tests or upload files to other platforms from my PC on the same office network, OpenMPTCPRouter splits the bandwidth across both connections flawlessly, saturating the full 100 Mbit/s.

I suspect the issue might be related to my port forwarding configuration for Plex on port 32400. Here’s my current setup:

<img width="500" alt="Config Screenshot 1" src="https://github.com/user-attachments/assets/04fb9814-6433-40b2-a562-1f22f538cf2d">
<img width="500" alt="Config Screenshot 2" src="https://github.com/user-attachments/assets/fd34d2e2-0c91-4574-8533-b588b9833836">

And here’s the Plex configuration, which appears to be fine:

<img width="600" alt="Plex Config Screenshot" src="https://github.com/user-attachments/assets/e1931084-90c7-496a-92aa-537ee60a7506">

Could there be something else I’m overlooking that might be causing this issue?

Thanks in advance for your help!

**Comment by TipsTricksMore** [comment]
@Ysurac do you got any idea regarding this?

**Comment by Ysurac** [comment]
I never tested plex, so I'm not sure how it work.
You can try to set another VPN in System->OpenMPTCProuter, "Wizard" tab, "Advanced settings" checkbox.

**Comment by TipsTricksMore** [comment]
Thank you for your help! I tried adjusting various VPN settings but, unfortunately, none seemed to resolve the issue at first.

After updating both my VPS and local router to the latest firmware, however, everything is now working smoothly! I also took the opportunity to reinstall my Plex Docker during this process.

I’m not entirely sure what caused the problem initially, but thankfully, it’s all sorted now.

Thanks again for your assistance @Ysurac !
